readme.md

```diff
@@ -73,6 +73,12 @@ Values: `start` `middle` `end`
 
 Position to truncate the string.
 
+##### space
+
+Type: `string`<br>
+Values: `left``right`
```

Comment:
This should just be a boolean, and when true:

- `foo ...`
- `... foo`
- `fo ... oo`

Reply:
Should this depend on the `position` option?

Reply:
Yes